gui/wxpython/lmgr/frame.py

```diff
@@ -304,6 +295,32 @@ def _createTabMenu(self):
 
         return menu
 
+    def _createDisplayPanel(self):
+        """Creates display panel"""
+        # create superior display panel
+        display_panel = wx.Panel(self.notebook, id=wx.ID_ANY)
+        # create display toolbar
+        dmgrToolbar = DisplayPanelToolbar(guiparent=display_panel,
+                                          parent=self)
+        # create display notebook
+        notebookLayers = GNotebook(parent=display_panel,
+                                   style=globalvar.FNPageStyle)
```

Comment:
Since you are changing this, can you use the Black formatting here (dmgrToolbar and notebookLayers)? It will result in this line staing the same after Black is applied globally possibly making navigating history easier. (I was going to suggest the same for DisplayPanelToolbar, but it seems it is actually just misplaced old code.)